### PR TITLE
Azure NextGen examples

### PR DESCRIPTION
**Pull request by mikhailshilkov**

### CODE REVIEW

File: azure-nextgen-py-appservice-docker/__main__.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+from pulumi_azure_nextgen.containerregistry import latest as containerregistry
+from pulumi_azure_nextgen.resources import latest as resources
+from pulumi_azure_nextgen.storage import latest as storage
+from pulumi_azure_nextgen.web import latest as web
+import pulumi_docker as docker
+import pulumi_random as random
+
+resource_group = resources.ResourceGroup("resourceGroup",
+    resource_group_name="appservice-docker-rg",
+    location="WestUS")
```
Comment (justinvp) [inline azure-nextgen-py-appservice-docker/__main__.py:11]: Should this use the location from the config?

File: azure-nextgen-py-appservice-docker/__main__.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
+import pulumi
```
Comment (justinvp) [inline azure-nextgen-py-appservice-docker/__main__.py:1]: Copyright header

```suggestion
# Copyright 2016-2020, Pulumi Corporation.  All rights reserved.

import pulumi
```

Applies throughout

File: azure-nextgen-py-appservice-docker/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+*.pyc
```
Comment (justinvp) [inline azure-nextgen-py-appservice-docker/.gitignore:1]:
```suggestion
*.pyc
venv/
```

File: azure-nextgen-py-appservice-docker/Pulumi.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+name: azureng-py-appservice-docker
+runtime: python
+description: Creates Azure App Service and Deploys Docker images to
+template:
+  config:
+    location:
+      description: Azure region to deploy resources to
+      secret: false
```
Comment (justinvp) [inline azure-nextgen-py-appservice-docker/Pulumi.yaml:8]: Should we add a default value?

```suggestion
      default: WestUS

```

File: azure-nextgen-py-appservice-docker/venv/pyvenv.cfg [hunk 5]
```diff
@@ -0,0 +1,3 @@
+home = /usr/local/bin
```
Comment (justinvp) [inline azure-nextgen-py-appservice-docker/venv/pyvenv.cfg:1]: Modify `.gitignore` to have `venv/` so this file isn't included.

File: azure-nextgen-ts-appservice-docker/README.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    $ az login
+    ```
+
+1. Set the azure location in which to run the test:
+    
+    ```
+    $ pulumi config set location westus2
+    ```
+
+1.  Run `pulumi up` to preview and deploy changes:
```
Comment (justinvp) [inline azure-nextgen-ts-appservice-docker/README.md:32]: Should we have an `npm install` step first?

File: azure-nextgen-py-appservice-docker/__main__.py (hunk 1 above)
Comment (mikhailshilkov) [reply azure-nextgen-py-appservice-docker/__main__.py:11]: Yes... I was certain I did that already... restoring.

**Pull request merged**